package.json

```diff
@@ -1,34 +1,29 @@
 {
-  "name": "toolhunt",
-  "version": "0.1.0",
+  "name": "vue-practice",
```

Comment:
We should probably rename this to the name of the project.

Reply:
Ouch, that was a real facepalm moment for me.  Fixed with [52f177c](https://github.com/wikimedia/toolhunt-ui/pull/5/commits/52f177cbcc186b8c550066e2eb71af70fd785bc1)